test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,60 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+     let svg = TestMethods.generateSVG(400, 100);
+      scale = new Plottable.Scales.Time();
+      scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
+      axis = new Plottable.Axes.Time(scale, "top");
+
+      function checkTierDisplayPosition (tierDisplayPositions: any[]) {
+        axis.tierLabelPositions(tierDisplayPositions);
+        axis.renderTo(svg);
+
+        let style = (<any>axis)._element.append("style");
+        style.attr("type", "text/css");
+        style.text(".plottable .axis.time-axis text { font-family: Arial; }");
```

Comment:
Can we add `!important` just in case
https://css-tricks.com/when-using-important-is-the-right-choice/